charge_lnd/policy.py

```diff
@@ -257,6 +257,9 @@ def match_by_chan(self, channel, config):
                     'min_htlcs_out', 'max_htlcs_out',
                     'min_sats_in', 'max_sats_in',
                     'min_sats_out', 'max_sats_out',
+                    'activity_period_exact',
```

Comment:
Can you please rename to `activity_period_ignore_channel_age`? I think this is more clear.